frontend/src/metabase/notifications/PulsesListSidebar.jsx

```diff
@@ -170,10 +172,29 @@ function buildFilterText(pulse, parameters) {
   }
 
   const [firstParameter, ...otherParameters] = activeParameters;
-  const numValues = [].concat(firstParameter.value).length;
-  const firstFilterText = `${firstParameter.name} is ${
-    numValues > 1 ? t`${numValues} selections` : firstParameter.value
-  }`;
+
+  // Format the first parameter value using the same logic as DefaultParametersSection
+  let formattedValue;
+  if (firstParameter.type && firstParameter.type.startsWith("date/")) {
+    const values = [].concat(firstParameter.value);
+    const formattedValues = values
+      .map((val) => formatDateValue(firstParameter, val))
+      .filter(Boolean);
```

Comment:
nit: it'd be more idiomatic to do `values = [...firstParameter.value]` to copy an array. But it this case, we could also just do the following since `map` and `filter` will give us a copy of the list:

```ts
const formatedValues = firstParameter.value
  .map((val) => formatDateValue(firstParameter, val))
  .filter(Boolean); 
```